### DISCUSSION TITLE
How to follow a link inside hover_doc?

### DISCUSSION THREAD

**Discussion by sorashi** [nvimdev/lspsaga.nvim#954] [Q&A] [answered]
I'm trying to navigate documentation. See image

![image](https://user-images.githubusercontent.com/6270283/226170266-2b5c8dbd-d47a-42f4-9f75-eb806e5393ea.png)

When I enter the `hover_doc` using <kbd>K</kbd><kbd>K</kbd>, I would like to follow the linked `enabledAnchorsCombinations`. I tried <kbd>Ctrl</kbd>+<kbd>]</kbd>, but it says

```
E433: No tags file
E426: tag not found: enabledAnchorsCombinations
```

I also tried <kbd>Enter</kbd>, <kbd>gd</kbd>, <kbd>gh</kbd>, and another <kbd>K</kbd>.

Am I missing something? Is it even possible?

**Comment by glepnir** [comment, marked as answer]
not possible now. can you create an issue for this i will support this open link in next version

**Reply by sorashi** [reply]
Ok, thank you :) #955
